src/utils/dateutils.py

```diff
@@ -8,3 +8,9 @@ def get_day_month(date: datetime) -> tuple:
 def get_current_day_month(utc_hour_offset: str) -> tuple:
     current_date = datetime.utcnow() + timedelta(hours=int(utc_hour_offset))
     return get_day_month(current_date)
+
+
+def get_years_difference_from_current_date(utc_hour_offset: str, date: str, date_format: str = '%Y-%m-%d') -> int:
+    date = datetime.strptime(date, date_format)
+    current_date = datetime.utcnow() + timedelta(hours=int(utc_hour_offset))
```

Comment:
Maybe we should take into consideration the utc-offset here as well

We need to consider that lambdas run in GMT and We are trying to send messages to a  GMT-5 timezone